lib/kaname/resource.rb

```diff
@@ -5,17 +5,16 @@ class Resource
     class << self
       def yaml(filename = 'keystone.yml')
         if File.exists?(filename)
-          @_yaml = YAML.load_file(filename)
-          expand_all_tenants
+          @_yaml ||= expand_all_tenants(YAML.load_file(filename))
         end
       end
 
       private
 
-      def expand_all_tenants
+      def expand_all_tenants _yaml
```

Comment:
This is differentiation between `yaml  variable` and `yaml method`.
@_yaml is used under this policy, isn't it?